.konflux/scripts/subscription-manager-bro.sh

```diff
@@ -284,8 +289,8 @@ function assert_diff {
         echo '/var/cache(: |/)dnf'
     } >> "$var_exclusions"
 
-    if { diff --brief --recursive --no-dereference "$expected/var" "$actual/var" || true; } | \
-        grep -vEf "$var_exclusions" | { grep '.'; }; then
+    if { diff --brief --recursive --no-dereference "$expected/var" "$actual/var" || true; } \
+                                                                                            | grep -vEf "$var_exclusions" | { grep '.'; }; then
```

Comment:
[nit, ignorable] This indent looks weird but I don't mind.
![image](https://github.com/stackrox/collector/assets/537715/811fe8f7-9a09-415b-b0fb-64d1a21e8357)